service/src/test/java/uk/gov/hmcts/reform/fpl/service/translations/provider/TranslatableHearingFurtherEvidenceProviderTest.java

```diff
@@ -0,0 +1,272 @@
+                                    SupportingEvidenceBundle.builder()
+                                        .document(DOCUMENT_REFERENCE)
+                                        .build())))
+                                .build())
+                        ))
+                        .build(),
+                    SELECTED_ORDER_ID));
+
+        }
+
+        @Test
+        void testIfCollectionEmpty() {
+            assertThrows(IllegalArgumentException.class,
+                () -> underTest.provideSelectedItemDocument(CaseData.builder()
+                        .hearingFurtherEvidenceDocuments(null)
+                        .build(),
+                    SELECTED_ORDER_ID));
+
+        }
+    }
+
+    @Nested
+    class Accept {
+
+        @Test
+        void testIfMatchingInCollection() {
+
+            boolean actual = underTest.accept(CaseData.builder()
+                    .hearingFurtherEvidenceDocuments(List.of(
+                        element(HearingFurtherEvidenceBundle.builder()
+                            .supportingEvidenceBundle(List.of(element(SELECTED_ORDER_ID,
+                                SupportingEvidenceBundle.builder().build())))
+                            .build())
+                    )).build(),
+                SELECTED_ORDER_ID);
+
+            assertThat(actual).isTrue();
+
+        }
+
+        @Test
+        void testIfNotMatchingInCollectionEmpty() {
+            boolean actual = underTest.accept(CaseData.builder()
+                    .hearingFurtherEvidenceDocuments(List.of(element(HearingFurtherEvidenceBundle.builder().build())))
+                    .build(),
+                SELECTED_ORDER_ID);
+
+            assertThat(actual).isFalse();
+
+        }
+
+        @Test
+        void testIfNotMatchingInCollection() {
+            boolean actual = underTest.accept(CaseData.builder()
+                    .hearingFurtherEvidenceDocuments(List.of(element(HearingFurtherEvidenceBundle.builder()
+                        .supportingEvidenceBundle(List.of(element(UUID_1,
+                            SupportingEvidenceBundle.builder()
+                                .document(DOCUMENT_REFERENCE)
+                                .build())))
+                        .build())))
+                    .build(),
+                SELECTED_ORDER_ID);
+
+            assertThat(actual).isFalse();
+
+        }
+
+        @Test
+        void testIfCollectionEmpty() {
+            boolean actual = underTest.accept(CaseData.builder()
+                    .hearingFurtherEvidenceDocuments(null)
+                    .build(),
+                SELECTED_ORDER_ID);
+
+            assertThat(actual).isFalse();
+        }
+    }
+
+    @Nested
+    class ApplyTranslatedOrder {
+
+        @Test
+        void applyMatchedOrder() {
+
+            when(time.now()).thenReturn(NOW);
+
+            List<Element<HearingFurtherEvidenceBundle>> bundles = new java.util.ArrayList<>();
```

Comment:
doesn't need to be fully qualified import

Reply:
same for other places in this file

Reply:
👍